docs/script-api.md

```diff
@@ -2666,6 +2688,16 @@ Pop unique identifier from the stack. Put after the input.
 - [`nil win_image(int image, int width, int height)`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=win_image) &GuiDrawContext::win_image
 \
 Draw image to window.
+### `VanillaRenderContext`
+- [`nil draw_text(const string& text, float x, float y, float scale_x, float scale_y, Color color, int alignment, int fontstyle)`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=draw_text) &VanillaRenderContext::draw_text
+\
+Draw text using the built-in renderer. Use in combination with ON.RENDER_PRE_HUD or ON.RENDER_POST_HUD. See vanilla_rendering.lua in the example scripts.
+- [`tuple<float, float> draw_text_size(const string& text, float scale_x, float scale_y, int fontstyle)`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=draw_text_size) &VanillaRenderContext::draw_text_size
+\
+Measure the provided text using the built-in renderer
+- [`nil draw_texture(int texture_id, int row, int column, float render_at_x, float render_at_y, float render_width, float render_height, Color color)`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=draw_texture) &VanillaRenderContext::draw_texture
+\
+Draw a texture in screen coordinates `x`, `y` using the built-in renderer. Use in combination with ON.RENDER_PRE_HUD or ON.RENDER_POST_HUD
```

Comment:
This comment should probably just say `ON.RENDER_*` at this point since you added a million new events  :D